sklearn/ensemble/_bagging.py

```diff
@@ -202,14 +202,16 @@ def _parallel_build_estimators(
     return estimators, estimators_features
 
 
-def _parallel_predict_proba(estimators, estimators_features, X, n_classes):
+def _parallel_predict_proba(
+    estimators, estimators_features, X, n_classes, predict_params
```

Comment:
I'd call these arguments `params` since they're not only given to `predict`. Same for the `decision_function` method